app/views/personas/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Test Users</h1>
+
+    <% @personas.each do |persona| %>
+      <h2 class="govuk-heading-m">
+        <%= persona.first_name %>
+      </h2>
+      <strong class="govuk-tag govuk-tag--turquoise govuk-!-margin-bottom-5">
+        <%= t(".#{persona.first_name.downcase}.persona_type", default: "") %>
+      </strong>
+      <p class="govuk-body">
+        <%= t(".#{persona.first_name.downcase}.description", default: "") %>
+      </p>
+
+      <% if t(".#{persona.first_name.downcase}.roles", default: "").present? %>
+        <p class="govuk-body">
+          <%= t(".roles_include", persona_name: persona.first_name, default: "") %>
+        </p>
+        <ul class="govuk-list govuk-list--bullet">
+          <% t(".#{persona.first_name.downcase}.roles").each do |role| %>
+            <li><%= role %></li>
+          <% end %>
+        </ul>
+      <% end %>
+
+      <%= form_tag("/auth/developer/callback", method: "post", data: {turbo: false}) do %>
+        <%= hidden_field_tag "email", persona.email %>
+        <%= hidden_field_tag "first_name", persona.first_name %>
+        <%= hidden_field_tag "last_name", persona.last_name %>
+        <button type='submit' class="govuk-button govuk-!-margin-bottom-2">
```

Comment:
This needs to be double quoted I think

Reply:
Resolved in [9f8f3b0](https://github.com/DFE-Digital/itt-mentor-services/pull/12/commits/9f8f3b07860c747a02a99e58c3b84c360cae9354)